NAMESPACE

```diff
@@ -70,11 +71,13 @@ importFrom(dplyr,right_join)
 importFrom(grDevices,col2rgb)
 importFrom(grDevices,rgb)
 importFrom(jsonlite,fromJSON)
+importFrom(magrittr,"%>%")
 importFrom(nabor,knn)
 importFrom(plyr,rbind.fill)
 importFrom(rgl,as.mesh3d)
 importFrom(rgl,plot3d)
 importFrom(rgl,points3d)
+importFrom(rlang,.data)
```

Comment:
What is this for?

Reply:
OK I see https://cran.r-project.org/web/packages/dplyr/vignettes/programming.html#different-data-sets. I haven't used that before. `rlang` is also a `dplyr` dependency so no actual increase.